velox/exec/HashBuild.cpp

```diff
@@ -718,9 +717,12 @@ bool HashBuild::finishHashBuild() {
       if (spiller_ != nullptr) {
         spillStats += spiller_->stats();
 
-        stats_.spilledBytes += spillStats.spilledBytes;
-        stats_.spilledRows += spillStats.spilledRows;
-        stats_.spilledPartitions += spillStats.spilledPartitions;
+        {
+          auto lockedStats = stats_.wlock();
```

Comment:
nit: updateSpillStats()?

Reply:
Once again. It is only used once in this class.
Also, other operators might update spill stats differently?
I believe, if we want to change that, then a separate diff would be more prudent. This one is too big already.